plugins/node/opentelemetry-instrumentation-fastify/src/instrumentation.ts

```diff
@@ -271,6 +280,20 @@ export class FastifyInstrumentation extends InstrumentationBase {
         spanName,
         spanAttributes
       );
+
+      if (instrumentation.getConfig().requestHook) {
+        safeExecuteInTheMiddle(
+          () => instrumentation.getConfig().requestHook!(span, { request }),
+          e => {
+            if (e) {
+              diag.error('request hook failed', e);
```

Comment:
no need for this one if you added `instrumentation._diag.error('request hook failed', e);`